src/utils/storage/__tests__/local.ts

```diff
@@ -0,0 +1,31 @@
+import { getItem, setItem } from '../local'
```

Comment:
I'm surprised this passes. Normally I have to mock the localStorage.

Reply:
Somehow it's already mocked.